packages/hooks/src/useMT5SVGEligibleToMigrate.ts

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { useStore } from '@deriv/stores';
+import { Jurisdiction, getFormattedJurisdictionCode } from '@deriv/shared';
+
+const useMT5SVGEligibleToMigrate = () => {
+    const { client } = useStore();
+    const { mt5_login_list } = client;
+
+    const mt5_migration_config = React.useMemo(() => {
+        const svg_accounts_to_migrate = mt5_login_list.filter(
+            account => account.landing_company_short === Jurisdiction.SVG && !!account.eligible_to_migrate
+        );
+
+        const has_svg_accounts_to_migrate = !!svg_accounts_to_migrate.length;
+        const no_of_svg_accounts_to_migrate = svg_accounts_to_migrate.length;
```

Comment:
You can use type coercion for this variable as well.

Reply:
we need `no_of_svg_accounts_to_migrate` as a number. could you please show an example of what you meant? 🙏🏼

Reply:
parseInt(svg_accounts_to_migrate.length). It is based on the preference TBH.

Reply:
```suggestion
        const no_of_svg_accounts_to_migrate = Number(svg_accounts_to_migrate.length);

```
I think @hirad-deriv meant this, to strictly types into number. But I think both are already Number.